src/messages/agent.rs

```diff
@@ -229,16 +229,20 @@ pub struct BackendStateMessage {
     /// The new state.
     pub state: BackendState,
 
+    /// The backend id.
+    pub backend: BackendId,
```

Comment:
Is it better to borrow this?

Reply:
No, it's fine like this. It's usually better for structs to own their members instead of holding references to them.

Reply:
I figured